### PR TITLE
Sort cluster by name is the settings

### PR DESCRIPTION
**Pull request by angelozerr**
Sort cluster by name is the settings

Signed-off-by: azerr <azerr@redhat.com>

### CODE REVIEW

**Comment by angelozerr** [comment]
This PR sort cluster name in the clusterSettings#getAll instead of sorting cluster in the kafka model (used only by explorer). It gives the capability to display cluster in wizard sorted by name.

Here a screenshot where abcd is created after my-cluster:

image

**Comment by fbricon** [comment]
please update the readme

**Comment by angelozerr** [comment]
> please update the readme

I have updated the README, please tell me if it's OK.

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
 Work with Kafka directly in Visual Studio Code. Kafka clusters running version 0.11 or higher are supported.
 
 Features:
-- Connect to multiple clusters
+- Connect to multiple clusters (clusters are displayed in explorer sorted by name)
```
[the remark this answers is not in the document]
Comment (fbricon) [reply README.md]: the user-visible change is that clusters are now sorted in the cluster selection wizard

[the remark this answers is not in the document]
Comment (angelozerr) [reply README.md]: ok thanks for the clarification. fixed

**Comment by fbricon** [comment]
Thanks @angelozerr!

**Pull request merged**